src/Core/Services/ConfigService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bit.Core.Abstractions;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Domain;
+using Bit.Core.Models.Response;
+using Bit.Core.Models.View;
+
+namespace Bit.Core.Services
+{
+    public class ConfigService : IConfigService
```

Comment:
This service should follow `server`'s implementation of [feature flags](https://github.com/bitwarden/server/blob/master/src/Core/Services/IFeatureService.cs), exposing the different flag variant methods (`bool`, `string` and `int`)

Reply:
Since i've already extended the config service in #2454 i'll add the variants there to avoid merge conflicts.

Reply:
Updated #2454 with variants.